pkgs/applications/networking/cluster/coop-cloud/abra.nix

```diff
@@ -0,0 +1,26 @@
+{buildGoModule, fetchFromGitea, lib}:
+buildGoModule rec {
+  pname = "abra";
+  version = "0.4.1-alpha";
+  src = fetchFromGitea {
+    domain = "git.coopcloud.tech";
+    owner = "coop-cloud";
+    repo = "abra";
+    rev = "4ba15df9b702fdeec4cc2089a826da7ef69fd711";
+    sha256 = "sha256-HSAWnwxMsX09PdPy2AyQGprV1fRN+lG+a67L+xd3GlA=";
+  };
+  vendorSha256 = "sha256-2o7GPEStjdBEY2RMhApMTPjrC5H/B9c3s1B2t2pTh+A=";
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Commit=${src.rev}"
+    "-X main.Version=${version}"
+  ];
+  meta = {
+    description = "The Co-op Cloud command-line utility belt 🎩🐇";
+    homepage = "https://coopcloud.tech";
+    downloadPage = "https://coopcloud.tech/abra";
+    license = lib.licenses.gpl3Plus;
+    maintaines = [lib.maintainers.akshaymankar];
```

Comment:
Typo: "maintainers"

(CI is currently red on this PR because of this error.)

Also nit: `maintainers = with lib.maintainers; [ akshaymankar ];` is the more idiomatic way. I usually `with lib;` the whole `meta` block too, but that's just personal preference.